lib/api/client-commands/frame.js

```diff
@@ -40,14 +41,20 @@ const findElement = function(selector) {
   });
 };
 
-module.exports = class Session extends ProtocolAction {
+class Frame extends ClientCommand {
   async command(frameId, callback) {
     if (arguments.length === 1 && typeof frameId === 'function') {
       callback = frameId;
 
       return this.transportActions.switchToFrame(callback);
     }
 
+    //Check if frameId is an HTML element in DOM and convert it into its CSS selector representation as frameId can only be of type number,null or string.
+
+    if (frameId instanceof Element) {
+      frameId = frameId.selector;
+    }
```

Comment:
`switchTo.Frame` also supports instance of WebElement. Please verify if this is needed